x-pack/plugins/cloud_security_posture/server/fleet_integration/fleet_integration.ts

```diff
@@ -46,7 +46,10 @@ export const onPackagePolicyPostCreateCallback = async (
   }
   // Create csp-rules from the generic asset
   const existingRuleTemplates: SavedObjectsFindResponse<CloudSecurityPostureRuleTemplateSchema> =
-    await savedObjectsClient.find({ type: cloudSecurityPostureRuleTemplateSavedObjectType });
+    await savedObjectsClient.find({
+      type: cloudSecurityPostureRuleTemplateSavedObjectType,
+      perPage: 1000,
```

Comment:
Since we'll have a single integration with all the rule templates, 1000 is a number of rules we'll get to one day.
worth thinking if we want a bigger number or a way to remember fixing it when the time comes.

Reply:
By the saved objects api [doc](https://www.elastic.co/guide/en/kibana/master/saved-objects-api-find.html), there is no other way to fetch all, I increased the number to 10,000. 

I'll also document there we need an integration test that validate that this number is higher than the rules templates in the integration repo.